src/Compute/Compute/Generated/VirtualMachineScaleSet/Config/SetAzureRmVmssSecurityProfileCommand.cs

```diff
@@ -47,7 +48,8 @@ public partial class SetAzureRmVmssSecurityProfileCommand : Microsoft.Azure.Comm
             Mandatory = false,
             Position = 1,
             ValueFromPipelineByPropertyName = true)]
-        [PSArgumentCompleter("TrustedLaunch", "ConfidentialVM")]
+        [ValidateSet(ValidateSetValues.TrustedLaunch, ValidateSetValues.ConfidentialVM, ValidateSetValues.Standard, IgnoreCase = true)]
```

Comment:
It's a breaking change. Can you hold this PR to next major release?

Reply:
@YanaXu , I'll just remove this and save it for later.